src/assets/stylesheets/shared/_layout.scss

```diff
@@ -585,3 +585,35 @@ $wizard-footer-color: $color-pf-black-200;
 
 }
 
+span,div
+{
+  &.wizard-status-success{
+    color:lime;
```

Comment:
@mindreeper2420 please make an issue for this, since it won't get fixed this week.